apache/superset
# fix: Center each import icon and add a tooltip

### SUMMARY
I have added a tooltip with text "Import [object]" (example: "Import Database", "Import Dataset", etc.) on all occurrences of the import icon: Database, Dataset, Chart, Dashboard, Saved Query. I have also centered each tooltip icon.

### BEFORE/AFTER SCREENSHOTS OR ANIMATED GIF

#### BEFORE:
<img width="386" alt="import-icon-before" src="https://user-images.githubusercontent.com/55605634/114792120-5dff7f00-9d4d-11eb-9197-ad697d4a8e61.png">

#### AFTER:
<img width="319" alt="import-icon-after" src="https://user-images.githubusercontent.com/55605634/114792148-6657ba00-9d4d-11eb-897d-25799d6d283c.png">


### TEST PLAN
- Navigate to all pages with an import icon (Database, Dataset, Chart, Dashboard, Saved Query)
- Observe that the icon is centered with it's neighboring button(s)
- Hover mouse pointer over the import icon
- Observe the appropriate relative "Import [object]" tooltip

### ADDITIONAL INFORMATION
- [x] Has associated issue: https://github.com/apache/superset/issues/13920
